compiler-cli/src/new/tests.rs

```diff
@@ -33,6 +33,54 @@ fn new() {
     assert!(toml.contains("name = \"my_project\""));
 }
 
+#[test]
+fn new_with_default_template() {
+    let tmp = tempfile::tempdir().unwrap();
+    let path = Utf8PathBuf::from_path_buf(tmp.path().join("my_project")).expect("Non Utf8 Path");
+
+    let creator = super::Creator::new(
+        super::NewOptions {
+            project_root: path.to_string(),
+            template: super::Template::Erlang,
+            name: None,
+            skip_git: false,
+            skip_github: true,
+        },
+        "1.0.0-gleam",
+    )
+    .unwrap();
+    creator.run().unwrap();
+
+    assert!(path.join("gleam.toml").exists());
+
+    let toml = crate::fs::read(path.join("gleam.toml")).unwrap();
+    assert!(!toml.contains("target = \"javascript\""));
```

Comment:
Use snapshots for the tests please

Reply:
Thanks for the hint, learning something new 👌 
I've set up snapshot assertions for those two tests in 4d18cd55f243c7517eab17e6482be5ae60ee76a2 .
Let me know if they need more work or if something is missing.